azure-container-apps-maven-plugin/src/main/java/com/microsoft/azure/maven/containerapps/parser/ConfigParser.java

```diff
@@ -30,22 +39,31 @@ public ContainerAppConfig getContainerAppConfig() {
         config.setAppName(mojo.getAppName());
         config.setResourceConfiguration(getResourceConfigurationFromContainers(mojo.getContainers()));
         config.setIngressConfig(getIngressConfig(mojo.getIngress()));
-        config.setImageConfig(getImageConfigFromContainers(mojo.getContainers()));
+        config.setRegistryConfig(getRegistryConfig());
+        config.setImageConfig(getImageConfigFromContainers());
         config.setScaleConfig(mojo.getScale());
         return config;
     }
 
 
     //todo(ruoyuwang): whether we should support multi containers? CLI/Portal only support one container
     //todo(ruoyuwang): support container registry part
-    public ContainerAppDraft.ImageConfig getImageConfigFromContainers(List<AppContainerMavenConfig> containers) {
+    public ContainerAppDraft.ImageConfig getImageConfigFromContainers() {
+        List<AppContainerMavenConfig> containers = mojo.getContainers();
         if (containers == null || containers.isEmpty()) {
             return null;
         }
-        final ContainerAppDraft.ImageConfig imageConfig = new ContainerAppDraft.ImageConfig(containers.get(0).getImage());
+        final String defaultImageName = String.format("%s%s/%s:%s", mojo.getRegistry().getRegistryName(), ContainerRegistry.ACR_IMAGE_SUFFIX, mojo.getAppName(), timestamp);
```

Comment:
To make it working for sovereign clouds, we need to read the suffix from somewhere.

Reply:
Just curious, does IntelliJ plugin follows the same image tagging naming convention?

Reply:
Will support sovereign clouds later. Now Intellij plugin also use this public cloud suffix and use appName:aca{timestamp} as the image tagging name. Here is an example: ruoyutest.azurecr.io/hello:aca-20241022135134356